lib/Elastica/JSON.php

```diff
@@ -14,11 +14,13 @@ class JSON
      * @link http://php.net/manual/en/function.json-decode.php
      * @link http://php.net/manual/en/function.json-last-error.php
      *
-     * @param string $json JSON string to parse
+     * @param string $args,... JSON string to parse
+     *
+     * @throws JSONParseException
      *
      * @return array PHP array representation of JSON string
      */
-    public static function parse(/* inherit from json_decode */)
+    public static function parse($args/* inherit from json_decode */)
```

Comment:
What is the benefit of having $args here? We fetch / overwrite them on line 24 anyways